internal/auth/repository/eventsourcing/handler/user_session.go

```diff
@@ -203,35 +208,52 @@ func sessionColumns(event eventstore.Event, columns ...handler.Column) ([]handle
 	}, columns...), nil
 }
 
+func (u *UserSession) sessionColumnsActivate(event eventstore.Event, columns ...handler.Column) ([]handler.Column, error) {
+	sessionID, err := u.idGenerator.Next()
+	if err != nil {
+		return nil, err
+	}
+	columns = slices.Grow(columns, 2)
+	columns = append(columns,
+		handler.NewCol(view_model.UserSessionKeyState, domain.UserSessionStateActive),
```

Comment:
Does this not create problems if you set a sessions active if it was already terminated?

Reply:
the session in the login UI can already be reactivated. there's also no specific creation event as with the session v2. until now the state column was handled in the case switch, since we also might need to update the id i created the `sessionColumnsActivate` function